jenkins-integration/jenkins-jobs/scenarios/puppetserver-infinite/JobDSL.groovy

```diff
@@ -0,0 +1,6 @@
+job.parameters {
+    stringParam('NUMBER_OF_HOURS', '2', 'Length of time to run the gatling simulation.')
+    choiceParam('CATALOG_SIZE', ['MEDIUM', 'EMPTY'], 'Catalog to use in simulation.')
+    // NODE_COUNT is offered as a list because gatling will not handle values that don't divide 1800 evenly (multiples of 300 also seem fine, which is why 1200 and 1500 are also there).
```

Comment:
I was thinking more in the description, as just a comment to the operator about why they can't pick arbitrary numbers. All we have to say is that Gatling won't distribute the load evenly for other options.

Reply:
ah got it